src/spas/libs/member-management/feature/src/lib/member-management/users/users.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from '../user-profile';
+import { UserService } from '../user.service';
+
+@Component({
+  selector: 'jwadmin-users',
+  templateUrl: './users.component.html',
+  styleUrls: ['./users.component.scss']
+})
+export class UsersComponent implements OnInit {
+
+  users: User[];
+  selectedUser: User;
+
+  constructor(private userService: UserService) { }
+
+  ngOnInit() {
+    this.getUsers();
+  }
+
+  onSelect(user: User) {
+    this.selectedUser = user;
+  }
+
+  getUsers(): void {
+    this.userService.getUsers()
+        .subscribe(users => this.users = users);
```

Comment:
will need to change to something like this:
`this.users=this.userService.getUsers();`